src/TodoApp/components/Title/TitleComponent.jsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+const Title = styled.div`
+	font-size: 4rem;
+	font-weight: 700;
+	color: white;
+	background: #5795f4;
+
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height: 200px;
+`;
+
+export const TitleComponent = () => {
```

Comment:
Seems unnecessary to have this component at all, because `Title` is already a component with the styles you need.